src/OrbitGl/CaptureViewElement.h

```diff
@@ -137,6 +137,8 @@ class CaptureViewElement : public Pickable, public AccessibleInterfaceProvider {
   float width_ = 0.;
   Vec2 pos_ = Vec2(0, 0);
   CaptureViewElement* parent_;
+
+  friend class CaptureViewElementTester;
```

Comment:
I don't have any issue of making a testing class friend of its element, but @beckerhe might have a different opinion.